components/ExpenseReporter/ExpenseReporter.stories.tsx

```diff
@@ -0,0 +1,18 @@
+import { ApolloProvider } from '@apollo/client'
+import React from 'react'
+
+import GetApolloClient from '../../apis/apollo.client'
+import ExpenseReporter from '.'
+
+export default {
+  title: 'ExpenseReporter',
+  component: ExpenseReporter,
+}
+
+const Template = () => (
+  <ApolloProvider client={GetApolloClient()}>
```

Comment:
apolloclient should be provided only once at the app.tsx level and consumed at all other components using useQuery wherever needed

Reply:
in storybook not aware of a way to provide wrapper classses yet. suggestions?

Reply:
oh this is stories file. Fine to keep it this way